src/sourceMaps/sourceMap.ts

```diff
@@ -161,6 +161,7 @@ export class SourceMap {
      * Will return null instead of a mapping on the next line (different from generatedPositionFor).
      */
     public authoredPositionFor(line: number, column: number): MappedPosition {
+        if (global['evalSources'] && global['evalSources'].find(s => this._generatedPath == s) && line > 0) line--; // check if eval code
```

Comment:
I think when code is an eval script, then `(function(){\n` is added and chrome debugger result is offset by 1 line vs source-map which use the script code.